test/fakeintake/server/server.go

```diff
@@ -367,6 +367,23 @@ func (fi *Server) handleDatadogRequest(w http.ResponseWriter, req *http.Request)
 	writeHTTPResponse(w, response)
 }
 
+func redactHeader(header http.Header) http.Header {
```

Comment:
For the record you could also use the agent scrubber, it is defined in its own modules and has no dependencies